package.json

```diff
@@ -5,6 +5,7 @@
     "packages/*"
   ],
   "dependencies": {
+    "gestalt-datepicker": "^33.4.3",
```

Comment:
@ponori could we 
* revert the changes to `package.json` and `yarn.lock`?
* remove the `package-lock.json` file? We use `yarn` so that should not show up

Reply:
Ugh, sorry for that. I'll make that fix. Thanks!